Course Material/Introduction to Quantum Computing and FiQCI/00 Superposition and measurement.ipynb

```diff
@@ -0,0 +1,682 @@
+    "+ Simulator: Quantum job is simulated on classical computer. Simulator can be ideal, or it can have a noise profile mimicking real quantum device\n",
+    "+ QPU (Quantum Processing unit): Quantum job is run on a real quantum computer\n",
+    "\n",
+    "In this notebook, we demonstrate both of these backends.\n",
+    "\n",
+    "First we choose is a simulator that performs calculations using statevectors. The following cell retrieves backend for the ideal, noiseless simulator from **qiskit.Aer**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "backend_simulator = Aer.get_backend('statevector_simulator')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To run jobs on real quantum computers, we need a different backend. \n",
+    "\n",
+    "We will use Helmi, the Finnish quantum computer connected to LUMI-supercomputer. Note, that to access Helmi you must be using correct node of LUMI.\n",
+    "\n",
+    "Following steps are needed:\n",
+    "+ Get environment variable **HELMI_CORTEX_URL**\n",
+    "+ Initialize **IQMProvider** with Helmi server URL\n",
+    "+ Retrieve backend for Helmi quantum computer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "HELMI_CORTEX_URL = os.getenv('HELMI_CORTEX_URL')\n",
+    "provider = IQMProvider(HELMI_CORTEX_URL)\n",
+    "backend_helmi = provider.get_backend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Third step: Allocate qubits\n",
+    "\n",
+    "\n",
+    "In our example, we need to allocate only one qubit\n",
+    "\n",
+    "We need to:\n",
+    "+ Create **QuantumCircuit** object\n",
+    "+ define the number of qubits and classical bits in our circuit\n",
+    "\n",
+    "The following cell allocates (creates) one qubit and one classical bit to our circuit:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qc = QuantumCircuit(1,1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can visualize the circuit by calling **draw()**. The **'mpl'** sets the matplotlib style renderer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">     \n",
+       "  q: \n",
+       "     \n",
+       "c: 1/\n",
+       "     </pre>"
+      ],
+      "text/plain": [
+       "     \n",
+       "  q: \n",
+       "     \n",
+       "c: 1/\n",
+       "     "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "qc.draw()"
```

Comment:
```suggestion
    "qc.draw("mpl")"
```

Additionally @TheKharleeci  can you make sure `pylatexenc` is installed on LUMI's Helmi's qiskit module.